### PR TITLE
feat: added text auto focus option

### PR DESCRIPTION
**Pull request by jinwoo-kim-nhn**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added/updated (for bug fixes/features)
- [x] It does not introduce a breaking change or has description for the breaking change

### Description

* addText 에 default값이 true인 autofocus옵션 추가.
* autofocus옵션이 true이면 오토포커싱 시키고 텍스트가 전체 선택되도록함

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: src/js/component/text.js [hunk 1]
```diff
@@ -202,6 +202,10 @@ class Text extends Component {
                 styles = snippet.extend(styles, options.styles);
             }
 
+            if (!snippet.isExisty(options.autofocus)) {
```
Comment (dongsik-yoo) [inline src/js/component/text.js:205]: `autofocus`가 false인 경우는 오토포커스가 안 가도록 되나요? `isExisty`라서 false면 여길 안 탈거 같아서요.

Comment (dongsik-yoo) [reply src/js/component/text.js:205]: 아 없으면 true라서 정상동작하겠네요. 위에 코멘트 무시해 주세요.

File: src/js/imageEditor.js [hunk 2]
```diff
@@ -995,6 +995,7 @@ class ImageEditor {
      *         @param {string} [options.styles.textAlign] Type of text align (left / center / right)
      *         @param {string} [options.styles.textDecoration] Type of line (underline / line-through / overline)
      *     @param {{x: number, y: number}} [options.position] - Initial position
+     *     @param {boolean} [options.autofocus] - text autofucus, default is true
```
Comment (dongsik-yoo) [inline src/js/imageEditor.js:998]: 오타 있어요. `autofucus`

Comment (jinwoo-kim-nhn) [reply src/js/imageEditor.js:998]: 감사합니다. 수정하겠습니다.

**Pull request merged**